src/test/java/com/flightstats/hub/webhook/WebhookTest.java

```diff
@@ -1,32 +1,38 @@
 package com.flightstats.hub.webhook;
 
+import com.flightstats.hub.dao.aws.ContentRetriever;
 import com.flightstats.hub.model.ContentKey;
 import com.flightstats.hub.model.MinutePath;
-import java.util.Optional;
-
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.Optional;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 class WebhookTest {
 
     private static final int CALLBACK_TIMEOUT_DEFAULT_IN_SEC = 120;
+    @Mock
+    private ContentRetriever contentRetriever;
     private Webhook webhook;
 
     @BeforeEach
     void setUp() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
Does this do the same thing as adding an annotation to the class? If so, is there an advantage to doing it one way or the other?